include/CppUTest/UtestMacros.h

```diff
@@ -232,6 +232,18 @@
 #define BYTES_EQUAL_TEXT(expected, actual, text)\
     LONGS_EQUAL_TEXT((expected) & 0xff, (actual) & 0xff, text)
```

Comment:
After the changes with the brackets, it could become UNSIGNED_LONGS_EQUAL_TEXT(). It would be easier to do this after the brackets thingie.